packages/dashboard/src/testing/testing-ground/testing-ground.tsx

```diff
@@ -1,13 +1,104 @@
 import { Component, State, h } from '@stencil/core';
 import { DashboardConfiguration } from '../../types';
 import { dashboardConfig } from '../mocks';
+import { XAnnotation } from '@synchro-charts/core';
+import { addXAnnotation, editXAnnotation, deleteXAnnotation } from '../../dashboard-actions/annotations';
+import { viewportEndDate, viewportStartDate } from '@iot-app-kit/core';
+
+const DEFAULT_LIST_VISIBLE_ANNOTATIONS = false;
 
 @Component({
   tag: 'testing-ground',
   styleUrl: 'testing-ground.css',
 })
 export class TestingGround {
   @State() dashboardConfiguration: DashboardConfiguration = dashboardConfig;
+  @State() selectedWidgetId = this.dashboardConfiguration.widgets[0].id;
+  @State() onlyListVisibleAnnotations = DEFAULT_LIST_VISIBLE_ANNOTATIONS;
+
+  onListVisibleAnnotations = (e: Event) => {
+    this.onlyListVisibleAnnotations = (e as any).target.checked;
```

Comment:
Need to remove all `(e as any)` in added lines before merge

Reply:
Also if you look at why checks are failing, it's because ESLint is finding too many warnings like `Warning:   111:44  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any`, so I think removing these will results in checks passing.

Reply:
Removed all cases of any!